broker/lib/cf/TokenInfo.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
```

Comment:
i feel we could probably move the module `cf` to common. I see scheduler also split from broker down the line and scheduler uses `cf` so it would be better to move this to common.

Reply:
So there is some code that will be used by all the bosh jobs (for ex. config.js), we put that in common, it will have minimal files.
Some code that is gonna be used by only few jobs and not all (Like cf, fabrikStatusPoller), used by only broker and scheduler.
I suggest we can have `cf` also on same level as `common` rather than inside `common` when we modularise `broker/` further.

Reply:
as discussed these common modules can move at root level as they can probably be used by other modules on further refactoring of broker code, for now this is good.